src/studio/src/designer/frontend/packages/ux-editor/src/containers/EditContainer.tsx

```diff
@@ -252,164 +119,75 @@ export function EditContainer(props: IEditContainerProvidedProps) {
-                    ? classes.active
-                    : props.component.type === 'Group'
-                    ? 'formGroup'
-                    : classes.formComponent
-                }
-                onClick={handleSetActive}
-                tabIndex={0}
-                onKeyPress={handleKeyPress}
-                component='div'
-              >
-                {isEditMode && component ? (
-                  <Grid item={true} xs={12} className={classes.activeWrapper}>
-                    <EditModalContent
-                      component={JSON.parse(JSON.stringify(component))}
-                      handleComponentUpdate={handleComponentUpdate}
-                    />
-                  </Grid>
-                ) : (
-                  <div className={`${classes.textPrimaryDark} ${classes.formComponentTitle}`}>
-                    <i
-                      className={`${classes.icon} ${
-                        componentIcons[component.type] || 'fa fa-help-circle'
-                      }`}
-                    />
-                    {component.textResourceBindings?.title
-                      ? truncate(
-                          getTextResource(component.textResourceBindings.title, textResources),
-                          80
-                        )
-                      : getComponentTitleByComponentType(component.type, language) ||
-                        getLanguageFromKey('ux_editor.component_unknown', language)}
-                  </div>
-                )}
-              </ListItem>
-            </div>
-          </Grid>
-          {!isEditMode && (
-            <Grid item={true} xs={1}>
-              <Grid container={true} direction='row' className={btnGrid()}>
-                <Grid item={true} xs={12}>
-                  {(activeListIndex === 0 || activeList.length < 1) && (
-                    <IconButton
-                      type='button'
-                      className={`${classes.formComponentsBtn} ${classes.specialBtn}`}
-                      onClick={handleComponentDelete}
-                      tabIndex={0}
-                    >
-                      <i className='fa fa-circletrash' />
-                    </IconButton>
-                  )}
-                </Grid>
-                <Grid item={true} xs={12}>
-                  {(activeList.length < 1 ||
-                    (activeList.length === 1 && activeListIndex === 0)) && (
-                    <IconButton
-                      data-testid='EditContainer-edit-button'
-                      type='button'
-                      className={classes.formComponentsBtn}
-                      onClick={handleOpenEdit}
-                      tabIndex={0}
-                    >
-                      <i className='fa fa-edit' />
-                    </IconButton>
-                  )}
-                </Grid>
-              </Grid>
-            </Grid>
+    <div className={cn(classes.wrapper, isEditMode && classes.editMode)}>
+      <div
```

Comment:
slike aktive handlinger bør ligge på knapper eller andre elementer som har dette native.

Reply:
Jeg er enig, men jeg ville ikke rote for mye i det som var der fra før. Så lenge man har en `tabIndex` skal ikke det være et problem. Tar uansett en nærmere titt på det i neste omgang.